File: pset1/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int altura;
    int fila;
    int columna;

    do
    {
        altura = get_int("ingrese la altura de la piramide:(un numero entero del 1 al 23)\n");
    }
    while (altura < 0 || altura > 23);

    for (fila = 1; fila <= altura; fila++)
    {
        for (columna = 1; columna <= (altura - fila); columna++)
        {
            printf(" ");
        }
        for (columna = 0; columna <= fila; columna++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/sentimental/mario/mario.py
from cs50 import get_int


def main():
    altura = -1
    while altura < 0 or altura > 23:
        altura = get_int("ingrese la altura de la piramide:(un numero entero del 1 al 23): ")

    for fila in range(1, altura+1, 1):
        for columna in range(1, (altura - fila + 1), 1):
            print(" ", end="")
        for columna in range(0, fila + 1, 1):
            print("#", end="")
        print()


if __name__ == "__main__":
    main()
